docs/how_to_guides/configuring_metadata_stores/how_to_configure_a_validation_result_store_in_s3.rst

```diff
@@ -3,16 +3,85 @@
+    - Identified the S3 bucket and prefix where Validation results will be stored.
 
-This guide is a stub. We all know that it will be useful, but no one has made time to write it yet.
+Steps
+-----
+1. **Identify your Data Context Validations Store**
 
-If it would be useful to you, please comment with a +1 and feel free to add any suggestions or questions below.
+    Look for the following section in your Data Context's ``great_expectations.yml`` file:
+
+    .. code-block:: yaml
+
+        validations_store_name: validations_store
+
+        stores:
+            validations_store:
+                class_name: ValidationsStore
+                store_backend:
+                    class_name: TupleFilesystemStoreBackend
+                    base_directory: uncommitted/validations/
+
+    The configuration file tells Great Expectations to look for Validations in a store called ``validations_store``. It also creates a ``ValidationsStore`` called ``validations_store`` that is backed by a Filesystem and will store validations under the ``base_directory`` ``uncommitted/validations`` (the default).
+
+
+2. **Update your configuration file to include a new store for Validation results on S3.**
+
+    In the example below, the new store's name is set to ``validations_S3_store``, but it can be any name you like.  We also need to make some changes to the ``store_backend`` settings.  The ``class_name`` will be set to ``TupleS3StoreBackend``, ``bucket`` will be set to the address of your S3 bucket, and ``prefix`` will be set to the folder in your S3 bucket where Validation results will be located.  If you are also storing :ref:`Expectations in S3, <how_to_guides__configuring_metadata_stores__how_to_configure_an_expectation_store_in_amazon_s3>` please ensure that the ``prefix`` values are disjoint and one is not a substring of the other.
+
+
+    .. code-block:: yaml
+
+        validations_store_name: validations_S3_store
+
+        stores:
+            validations_S3_store:
+                class_name: ValidationsStore
+                store_backend:
+                    class_name: TupleS3StoreBackend
+                    bucket: '<your_s3_bucket_name>'
+                    prefix: '<your_s3_bucket_folder_name>'
+
+3. **Confirm that the new Validations store has been added by running ``great_expectations store list``.**
 
-If you want to be a real hero, we'd welcome a pull request. Please see :ref:`the Contributing tutorial <tutorials__contributing>` and :ref:`How to write a how to guide` to get started.
+    Notice the output contains two Validations Stores: the original ``validations_store`` on the local filesystem and the ``validations_S3_store`` we just configured.  This is ok, since Great Expectations will look for Validation results on the S3 bucket as long as we set the ``validations_store_name`` variable to ``validations_S3_store``.  Also, copy existing Validation results to the S3 bucket if necessary.
```

Comment:
"Copy existing Validation Results" seems like it should be its own (optional) step.